src/bci_build/package/__init__.py

```diff
@@ -179,6 +179,16 @@ def lifecycle_data_pkg(self) -> list[str]:
             return ["lifecycle-data-sle-module-development-tools"]
         return []
 
+    @property
+    def common_devel_packages(self) -> list[str]:
+        """Returns a list of common development packages that are needed for
+        all development containers"""
+        return (
+            ["findutils", "gawk", "git-core", "curl", "procps"] + ["util-linux"]
+            if self.value in (OsVersion.TUMBLEWEED.value,)
```

Comment:
This looks nicer imho

```suggestion
            if self.is_tumbleweed
```

Reply:
done